lib/ssh_scan/database/sqlite_db.rb

```diff
@@ -0,0 +1,65 @@
+require 'sqlite3'
+
+module   SSHScan
+  module Database
+    class SQLite
+      def initialize(database_name = "api.db")
+        if File.exist?(database_name)
+          @db = ::SQLite3::Database.open(database_name)
+        else
+          @db = ::SQLite3::Database.new(database_name)
+          create_schema
+        end
+      end
+
+      def create_schema
+        @db.execute <<-SQL
+          create table api_schema (
+            worker_id varchar(100),
+            uuid varchar(100),
```

Comment:
I'm wondering if in the schema we should have uuid, result, and then worker_id.  I say that because functionally the only reason we care about who did the work is if the work is somehow corrupted or problematic in some way, so I'm thinking of it more like an attribute rather than say a primary key item.